integration-cli/docker_cli_overlaynetwork_test.go

```diff
@@ -0,0 +1,34 @@
+//+build !windows
+
+package main
+
+import (
+	"strings"
+
+	"github.com/docker/docker/pkg/integration/checker"
+	"github.com/go-check/check"
+)
+
+func (s *DockerSwarmSuite) TestOverlayAttachable(c *check.C) {
+	d := s.AddDaemon(c, true, true)
+	s.AddDaemon(c, true, true)
+	s.AddDaemon(c, true, true)
```

Comment:
@mavenugo I guess it's to make sure it working with multiple node 👼. This will create 3 manager node though I think, we might want to create some worker node instead (for the 2 additional ones)